Applications/Utils/ModelPreparation/PartitionMesh/MeshPartitioning.h

```diff
@@ -0,0 +1,86 @@
+/*!
+  \file MeshPartitioning.h
+  \date   2016.05
+
+  \brief  Declare a class to perform mesh partitioning
+
+  \copyright
+  Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+             Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+
+*/
+
+#ifndef MESH_PARTITIONING_H_
+#define MESH_PARTITIONING_H_
+
+#include <vector>
+#include <string>
+#include <fstream>
+
+#include "MeshLib/Mesh.h"
+
+namespace MeshLib
```

Comment:
The file is in the directory ` Applications/Utils`, so the namespace shouldn't be `MeshLib`